src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java

```diff
@@ -56,6 +58,12 @@ public void run(String... topics) {
             this.jsonMsgList.put("depositRequests", jsonRequests);
             this.getRestDepositor().deposit(jsonMsgList.toString());
          }
+         try {
+            // add 1 second sleep between sending messages to SDX
+            TimeUnit.SECONDS.sleep(1);
```

Comment:
Looks like Brandon suggested moving away from this delay & implementing a batch processing solution instead.

Reply:
This has been removed